<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1876</width>
    <height>1146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyTrader v0.4</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>1611</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>잔고 및 보유종목현황</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>1741</width>
       <height>111</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>통화코드</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>외화예수금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>미수금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>선물청산손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>옵션청산손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>옵션결제차금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>선물평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>옵션평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>옵션평가차금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>증거금율</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>원화대용평가금액</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>예탁자산평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>익일예탁금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>위탁증거금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>미체결증거금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>포지션증거금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>유지증거금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추가증거금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수수료</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문가능금액</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>인출가능금액</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>위험도</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>1004</x>
       <y>537</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>-140</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>잔고확인</string>
     </property>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>61</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>예수금</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총매입</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총평가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총손익</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총수익률</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>추정자산</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>현금주문</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>530</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간 조회</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>180</y>
       <width>610</width>
       <height>341</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률 (%)</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>660</y>
      <width>630</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>자동 선정 종목 리스트</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>610</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>주문유형</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>호가구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>가격</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>상태</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>630</y>
      <width>381</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>48</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>48</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>48</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>244</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>48</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>48</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>48</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>134</width>
       <height>30</height>
      </rect>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>244</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>320</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>주문가능수량</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1876</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
